mhs/README.md

```diff
@@ -33,10 +33,10 @@ MHS takes a number of environment variables when it is run. These are:
 the most logs and `CRITICAL` displays the least. Note: Setting this value to one of the more detailed 'standard' Python
 log levels (such as `DEBUG` or `NOTSET`) may result in the libraries used by this application logging details that
 contain sensitive information such as the content of messages being sent.
-* `MHS_PARTY_KEY` The party key associated with your MHS.
-* `MHS_CLIENT_CERT` (outbound & Spine Route Lookup service only) Your endpoint certificate
-* `MHS_CLIENT_KEY` Your endpoint private key
-* `MHS_CA_CERTS` Should include the following in this order: endpoint certificate, endpoint issuing subCA certificate, root CA Certificate.
+* `MHS_SECRET_PARTY_KEY` The party key associated with your MHS.
+* `MHS_SECRET_CLIENT_CERT` (outbound & Spine Route Lookup service only) Your endpoint certificate
+* `MHS_SECRET_CLIENT_KEY` Your endpoint private key
```

Comment:
Probably worth highlighting that this is `(outbound & Spine Route Lookup service only)` too.

Reply:
Given we only use the client key with the client certificate, shouldn't we also have?:
```suggestion
* `MHS_SECRET_CLIENT_KEY` (outbound & Spine Route Lookup service only) Your endpoint private key
```

Reply:
AFAICT `MHS_CLIENT_KEY` is used in outbound, inbound and spine route lookup:
- https://github.com/nhsconnect/integration-adaptors/blob/f6c5e5cf63d3e9c7eca11d5e048af5f463d6d720/mhs/inbound/main.py#L61
- https://github.com/nhsconnect/integration-adaptors/blob/f6c5e5cf63d3e9c7eca11d5e048af5f463d6d720/mhs/outbound/main.py#L96
- https://github.com/nhsconnect/integration-adaptors/blob/f6c5e5cf63d3e9c7eca11d5e048af5f463d6d720/mhs/spineroutelookup/main.py#L41

Reply:
Discussed offline with @gareth-allan, realised that we probably should be passing `CLIENT_CERT` to MHS inbound, and at the same time be able to remove the endpoint certificate from `CA_CERTS`. I've made this change in 158d2bb.